itest/rust/src/runner.rs

```diff
@@ -33,10 +33,22 @@ impl IntegrationTests {
         gdscript_file_count: i64,
         allow_focus: bool,
         scene_tree: Gd<Node>,
+        filters: VariantArray,
     ) -> bool {
         println!("{}Run{} Godot integration tests...", FMT_CYAN_BOLD, FMT_END);
-
-        let (rust_tests, rust_file_count, focus_run) = super::collect_rust_tests();
+        let filters = filters
+            .iter_shared()
+            .filter_map(|x| x.try_to().ok())
```

Comment:
This silently skips invalid elements, possibly hiding logic errors. Don't do that -- just assume all arguments have the correct type. Anything else is a bug that must be fixed, not ignored.

TLDR: use `.map(|v| v.to::<String>())`